src/main/java/org/apache/commons/compress/archivers/zip/X000A_NTFS.java

```diff
@@ -97,13 +105,13 @@ private static FileTime zipToFileTime(final ZipEightByteInteger z) {
         }
         return TimeUtils.ntfsTimeToFileTime(z.getLongValue());
     }
-
```

Comment:
Keep this empty line.

Reply:
Removed by mistake. I'll take a look at the whole changelog again.